roles/nuage_node/vars/main.yaml

```diff
@@ -0,0 +1,10 @@
+---
+vrs_config: /etc/default/openvswitch
+uplink_interface: "{{ ansible_default_ipv4.interface }}"
+vsp_k8s_yaml: /usr/share/vsp-k8s/vsp-k8s.yaml
+config_dir: "{{ openshift.common.config_base }}/node"
+client_cert: "{{ config_dir }}/system:node:{{ openshift.common.hostname }}.crt"
+client_key: "{{ config_dir }}/system:node:{{ openshift.common.hostname }}.key"
```

Comment:
@abhat We weren't able to use the service account to access the local kubelet, but the service account was able to read from the API server just fine.